sources/packages/backend/libs/sims-db/src/entities/sfas-individual.model.ts

```diff
@@ -194,4 +194,21 @@ export class SFASIndividual extends BaseModel {
     nullable: false,
   })
   extractedAt: Date;
+  /**
+   * The most recent Part-time Master Student Loan Agreement Number (agreement_num.sail_msfaa_numbers).
+   */
+  @Column({
+    name: "part_time_msfaa_number",
+    nullable: true,
+  })
+  partTimeMSFAANumber?: string;
+  /**
+   * The most recent Part-time Master Student Loan Agreement effective date (effective_date.sail_msfaa_numbers).
+   */
+  @Column({
+    name: "part_time_msfaa_effective_date",
+    type: "date",
+    nullable: true,
+  })
+  partTimeMSFAAEffectiveDate?: Date;
```

Comment:
As per typeorm, it must be string right?

Reply:
I missed it too. Changed now.